tests/integration/api/v1/curated_recommendations/test_curated_recommendations.py

```diff
@@ -615,8 +615,8 @@ async def test_curated_recommendations_preferred_topic(self, preferred_topics):
             corpus_items = data["data"]
 
             assert response.status_code == 200
-            # assert total of 80 items returned
-            assert len(corpus_items) == 80
+            # assert items are returned
+            assert len(corpus_items) >= 80
```

Comment:
might need a refresher... but why the test case above `test_curated_recommendations`  asserts 100 corpus items and this test `test_curated_recommendations_utm_source` checks for >= 80? do we also need the `=`?

Reply:
Good point. `== 100` is more clear.
```suggestion
            assert len(corpus_items) == 100
```